### ISSUE TITLE
[Accessibility] Keyboard navigation issues

### ISSUE THREAD

**Issue by lpalmaro** [Polymer/polymer-element-catalog#104]
When I click on one of the tiles, e.g. Fe to start, there are keyboard/screen reader navigation issues on this page. I am unable to navigate to the top right buttons to switch the view, or to access the cart/ right panel. This all needs to be fully keyboard navigable, and towards the top of the tab order, similar to how it visually appears from left to right.

Tested: 
"Chrome OS / ChromeVox
Mac / VoiceOver
Win / NVDA

**Comment by azakus** [comment]
The element docs and sections should be `tabindex=0` to help the keyboard navigation.
